sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/DownloadManifestOptions.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.containers.containerregistry.models;
+
+/**
+ * Options for configuring the download manifest operation.
+ */
+public final class DownloadManifestOptions {
+    private String tag;
+    private String digest;
```

Comment:
Do any of these fields need validation?

Reply:
I have added null validation to the static methods but other than that no - not really.

We can try to add validation for digest as it is SHA but we have not done it any where else in the library so would avoid it here as well.